hooli_data_eng/assets/dbt_assets.py

```diff
@@ -100,12 +110,21 @@ def get_owners(self, dbt_resource_props: Mapping[str, Any]):
         ]
 
 
-class CustomDagsterDbtTranslatorForViews(CustomDagsterDbtTranslator):
-    def get_auto_materialize_policy(self, dbt_resource_props: Mapping[str, Any]):
-        return allow_outdated_and_missing_parents_policy
+class CustomDagsterDbtTranslatorForEager(CustomDagsterDbtTranslator):
```

Comment:
This pattern is reasonable, but a better option might be to just have a single translator which branches based on the model name, i.e.:

class CustomDagsterDbtTranslator(DagsterDbtTranslator):
        def get_automation_condition(self, dbt_resource_props: Mapping[str, Any]):
            model_name = dbt_resource_props[...]
            if model_name == "company_perf":
                return AutomationCondition.any_downstream_conditions()
            elif model_name == "...": ...